probe_scraper/probe_expiry_alert.py

```diff
@@ -128,7 +128,10 @@ def find_existing_bugs(
 
     probes_with_bugs = {}
     for bug in found_bugs:
-        if re.search(r"release: version (\d+)", bug["description"]).group(1) != version:
+        if (
+            re.search(r"release: \[?version (\d+)", bug["description"]).group(1)
```

Comment:
optionally match the bracket that now appears here https://github.com/mozilla/probe-scraper/blob/ade1d7c958e8a95af663f2992314beda50584fa0/probe_scraper/fog_checks.py#L79
but still does not appear here https://github.com/mozilla/probe-scraper/blob/ade1d7c958e8a95af663f2992314beda50584fa0/probe_scraper/probe_expiry_alert.py#L45